homework/week-0/src/nyc/c4q/JVidals1/PlayingCards.java

```diff
@@ -0,0 +1,59 @@
+package nyc.c4q.JVidals1;
+/**
+ * Created by s3a on 5/1/15.
+ * Janneisy Vidals
+ * 2.1 Accesscode
+ * HW Unit 1 - Problem 2  Cards
+ * Create an abstract class representing a game which includes a getpieces().
+ *
+ */
+
+import java.util.Collections;
+import java.util.ArrayList;
+
+public abstract class PlayingCards extends Game
+{
+    @Override
+    public boolean equals(Object obj)
+    {
+        return super.equals(obj);
+    }
+
+
+    public static boolean isSorted(Card[] list)
+    {
+        for(int i = 1; i < list.length; i++)
+        {
+            if(list[i - 1].compareTo(list[i]) > 0)
```

Comment:
Can just return value of if statement here instead of `return false` and `return true`.